examples/config-bolt-repodb.json

```diff
@@ -0,0 +1,25 @@
+{
+    "distSpecVersion": "1.0.1-dev",
+    "storage": {
+        "rootDirectory": "/tmp/zot",
+        "repoDBDriver": {
```

Comment:
we need to call this something else, don't know what yet!

Reply:
You mean "repoDB" in general, or the key names in the configuration?